src/Uno.UWP/Storage/Helpers/StorageFileHelper.cs

```diff
@@ -0,0 +1,14 @@
+﻿#nullable enable
+using System.Threading.Tasks;
+
+namespace Windows.Storage.Helpers;
+
+public partial class StorageFileHelper
+{
+	/// <summary>
+	/// Determines if a file exists within application package
+	/// </summary>
+	/// <param name="fileName">relative file path</param>
+	/// <returns>A task that will complete with a result of true if file exists, otherwise with a result of false.</returns>
+	public static async Task<bool> ExistsInPackage(string fileName) => await FileExistsInPackage(fileName);
```

Comment:
Let's also add an entry in the docs, at the end explaining what the API does and why:
https://github.com/unoplatform/uno/blob/master/doc/articles/features/file-management.md